docs/documentation/api/Server.md

```diff
@@ -24,6 +24,10 @@ A config object with the following options:
 
 3. `transport` (_object_): the transport implementation.
    If not provided, socket.io is used.
+   
+4. `generateCredentials` (_function_): an optional function that returns player credentials to store in the game metadata and validate against. If not specified, the Lobby’s `uuid` implementation will be used.
+
+5. `authenticateCredentials` (_function_): an optional custom function that tests if a player’s move is made with the correct credentials when using the default socket.io transport implementation.
```

Comment:
Let's call this "an optional function" (remove the word "custom") to be consistent with the line above.

Reply:
Resolved in d07c8b2